addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -406,23 +414,23 @@ export class HtmlField extends Component {
                 value = this._wrap(value);
             }
 
-            // this bug only appears on some computers with some chrome version.
-            let avoidDoubleLoad = 0;
+                        // this bug only appears on some computers with some chrome version.
+                        let avoidDoubleLoad = 0;
```

Comment:
Didn't we say we'd remove this ? I'm ok if you found a way to keep it, but I seem to remember you telling me that it didn't work and that you'd have to remove it ? I'm confused now... ^^'

Reply:
I remember us saying we should just have 2 different behaviors

I just check if we're in the sandboxedPreview mode, and if we are I resolve the promises and call the updates directly in `onload` instead of waiting for the script.

------------------

As a side-note, it seems like it's impossible to test `mass_mailing_html_field` properly because it uses the same "put script in the code of an iframe" approach to waiting for it to be rendered. Except that code is only executed in the real usecase because the iframe is reloaded afterwards via side-effects, like menus loading in. Since inserting script into a document doesn't actually run the script unless it's reloaded.

At least that's my assesment of the problem. Generally you can notice the one existing test doesn't actually get past `_loadIframe`, and in debug mode you just get an iframe with nothing in it. It's a bit annoying since we can't actually test the field, and it seems like there's a race-condition kind of thing we might want to get rid of at some point.

Reply:
I remember now that we said we'd keep the two behavors.

For the custom script inside the iframe thing, I might have an idea to make them executable without reload. However, I'm not sure I understand what script you are talking about specifically. Would you care linking me to those ?

I'm not sure I understand what race condition you are referring to, either.